<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotterDialog</class>
 <widget class="QDialog" name="PlotterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plotting from &lt;none&gt;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QPushButton" name="inUpdate">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="inDataSourceLine">
     <property name="minimumSize">
      <size>
       <width>489</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>C:\Users\shand.shand-PC\Workspace\best\daqmanager\fffff</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="inDataSource">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Data Source</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="MplWidget" name="mpl" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
  <action name="mplactionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="mplactionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
